src/main.cpp

```diff
@@ -2,15 +2,64 @@
 #include <seqan3/core/debug_stream.hpp>
 #include <seqan3/io/sam_file/all.hpp>
 
+#include <cereal/archives/binary.hpp>
+
 #include <bamit/all.hpp>
 
-struct CmdOptions
+struct IndexOptions
+{
+    std::filesystem::path input_path{};
+};
```

Comment:
If that is the only option we want to have, there is no need for an extra struct.